clients/client-python/gravitino/catalog/fileset_catalog.py

```diff
@@ -92,7 +92,7 @@ def create_fileset(
         self,
         ident: NameIdentifier,
         comment: str,
-        type: Catalog.Type,
+        type: Fileset.Type,
```

Comment:
Yes, you are right.

The corresponding java code in the here: https://github.com/datastrato/gravitino/blob/main/api/src/main/java/com/datastrato/gravitino/file/FilesetCatalog.java#L76